src/surface/mesh.cpp

```diff
@@ -96,52 +96,45 @@ namespace MR
     void Mesh::load_vtk (const std::string& path)
     {
 
-      std::ifstream in (path.c_str(), std::ios_base::in);
+      std::ifstream in (path.c_str(), std::ios_base::binary);
```

Comment:
```suggestion
      std::ifstream in (path.c_str(), std::ios_base::in | std::ios_base::binary);
```
Not sure what the behaviour will be if the path doesn't exist and only the `binary` flag is set; it might create an empty file?

Reply:
It took me a while to find out any info on what the flags end up being set to when non-default, but I finally found confirmation of the obvious [on the cplusplus website](https://cplusplus.com/reference/fstream/ifstream/ifstream/):

> * `in` is always set for [ifstream](https://cplusplus.com/ifstream) objects (even if explicitly not set in argument _mode_).

So there's no scope for that using a `std::ifstream`... :sweat_smile:

Reply:
Sorry, hadn't realised this was a suggested change - must be a new feature I hadn't come across. Happy to commit that change too if you think it's important to be explicit, though I personally feel this is redundant - up to you.

Reply:
Lol... ignore me.